projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -144,7 +146,13 @@ export class KeyNavigationGridController implements OnDestroy {
 
     const items = getTabableItems(activeCell);
     const item = activeCell.getAttribute('role') !== 'columnheader' && items[0] ? items[0] : activeCell;
-    item.focus();
+    if (scrollCell) {
+      item.focus();
+    } else {
+      item.focus({
+        preventScroll: true,
+      });
+    }
```

Comment:
This can be simplified to `item.focus({ preventScroll });`. I'd flip the boolean argument to make this work.